src/video-player.js

```diff
@@ -215,9 +214,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
 
     await this._initDash(options);
 
-    throttle(() => {
-      this.loadingLazyPlugins = false;
-    });
+    this.loadingLazyPlugins = false;
```

Comment:
I think enclosing this in a setTimeout is more explicit than what you did at the dashPlugin function - essentially per my understanding the issue is here, you want this to kinda of be executed in the next cycle, i think having the workaround closer to the issue makes it easier to understand and harder to just lose by mistake when refactoring/changing.

Reply:
It might be more clear, yes. but the effect here is not exactly the same.
This time I'm postponing the resolved state of the `dashPlugin` promise, instead of just postponing setting the `this.loadingLazyPlugins` value while keep running (which is what I did in the previous version).
To me this looks less prone to race-condition issues.